src/workloads/query/DensifyHours.yml

```diff
@@ -91,20 +91,36 @@ Actors:
   - Repeat: 1
   - *Nop
 
-- Name: DensifyHours
+- Name: IncreaseInternalQueryMaxAllowedDensifyDocs
   Type: RunCommand
   Threads: 1
   Phases:
   - *Nop
+  - Repeat: 1
+    Database: admin
+    Operation:
+      OperationMetricsName: internalQueryMaxAllowedDensifyDocs
+      OperationName: RunCommand
+      OperationCommand:
+        setParameter: 1
+        internalQueryMaxAllowedDensifyDocs: 10000000
+
+  - *Nop
```

Comment:
Regarding the Nop in between. Based on my understanding one should add 'Quiesce' phase/actor instead.

Reply:
+1 to the question why we need to increase densify limits?
Is it because before cursor was not exhausted and we didn't hit the limit?

About Quiesce phase: I would suggest having a separate query audit ticket where we add it to all workloads after loading data and before querying it. Also, I would like to have some data to confirm that having quiesce actor before reading actually makes the tests more stable, but again, I think it is a discussion for a separate ticket.